.circleci/config.yml

```diff
@@ -53,6 +53,7 @@ jobs:
           command: |
             mkdir /tmp/test-results
             ./cc-test-reporter before-build
+            openssl version
```

Comment:
is this just for debugging?

Reply:
Yeah, I think the OpenSSL version in the CircleCI image does not match with our usage and it's causing issues. While it's probably possible to figure it out and re-install/re-build in the test run, it's probably easier to move to GitLab in #394